invenio_rdm_records/fixtures/vocabularies.py

```diff
@@ -188,7 +178,17 @@ def load(self):
 
         Fixtures found later are ignored.
         """
-        self._loaded_vocabularies = set()
+        # Prime with existing (sub)vocabularies
```

Comment:
minor: I am not 100% I understand the comment 😅

Reply:
Ah, is it "prime" (any of the verb definition here: https://www.wordnik.com/words/prime)? Basically, we don't start with an empty set of loaded vocabularies. Instead we start with the vocabularies already present in the database, so that we don't run into integrity errors / we skip previously loaded vocabularies. Would no comment be better?

Reply:
Nope, leave the comment, is good. Just wanted to make sure that I understood it. Thanks 👼